safep/fileIO.py

```
@@ -145,118 +145,135 @@ def parse_Colvars_log(filename):
+def initialize_global_conf(lines):
+    """New colvars initialization
+
+    Args:
+        lines (list[str]): The config file
+
+    Returns:
+        dict: initialized colvars config dictionary
+
+    """
+    global_conf = {}
+    for line_number, line in enumerate(lines):
```

Ruff (0.15.20)

[warning] Loop control variable `line_number` not used within loop body

(B007)